README.md

```diff
@@ -123,6 +123,17 @@ Examples:
 
 `--verifier-id`: ID of entity verifying the policy for the VSA.
 
+#### Version
```

Comment:
The example source command is out of date as well https://github.com/liatrio/gh-trusted-builds-attestations#use